test/tests/steel_creep_damage_oh/tests

```diff
@@ -5,6 +5,7 @@
     type = 'Exodiff'
     input = 'steel_creep_damage.i'
     exodiff = 'steel_creep_damage_out.e'
+    valgrind = none
```

Comment:
Why no valgrind? Just slow?

Reply:
I guess you added a bunch of time steps, right? Does the test run fast enough in opt? It should be under 2 seconds.

Reply:
Yeah, the test takes longer with this change. I was afraid it was going to be too much. I can undo this.

Reply:
Not sure if we can have a meaningful test that runs under 2 seconds and has a more gradual loading. It takes about 10 seconds in `opt`

Reply:
@bwspenc added a shorter running test that will go through valgrind, even though it doesn't get to go through the entire physics of the problem. Does that work?

Reply:
Yeah, something like that should be fine. If the main test takes 10 seconds, it should be marked 'heavy'. Is the only difference between the two inputs the end time?  If so, probably best to just use cli_args. Maybe just have the standard short test with the early end time, and then a second variant with a later end time that is marked as heavy and skipped in valgrind.

Reply:
Did that. However, as this PR stands now, it means a regression in the capabilities that are being tested. There is only `heavy` testing of the code paths and physics that account for elements actually losing load carrying capacity due to damage.

As far I can tell, tests will pass without timing out with that test not marked as `heavy`.